.github/workflows/tox_verify.yaml

```
@@ -8,11 +8,12 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: ubuntu-24.04
-            python: "3.10"
-            evm-type: "stable"
-            tox-cmd: "uvx --with=tox-uv tox"  # run-parallel --parallel-no-spinner"  # TODO: disable parallelisation for uv testing
-          - os: ubuntu-24.04
+          # Temporarily disable
+          # - os: ubuntu-latest
+          #   python: "3.10"
+          #   evm-type: "stable"
+          #   tox-cmd: "uvx --with=tox-uv tox"  # run-parallel --parallel-no-spinner"  # TODO: disable parallelisation for uv testing
+          - os: ubuntu-latest
```

This will cause EELS to fail the RIPEMD160 precompile tests, we need to use `ubuntu-24.04`:
```suggestion
          - os: ubuntu-24.04
```

---

This is a typo :see_no_evil:

---

More info will be available in the docs with https://github.com/ethereum/execution-spec-tests/pull/847

---

Fixed.

---

Good typo! Build was green! They bumped `ubuntu-latest` to be `ubuntu-24.04` on 2024-09-25! :tada:

---

https://github.com/actions/runner-images/commit/ae99c16b0cf27c0cca7ef0b08e7bcf13c33f6cfa

---

Ah wait, should I bring it back then? :P